app/DynamicsAdapter/Fams3Adapter.Dynamics/Employment/SSG_Employment.cs

```diff
@@ -0,0 +1,82 @@
+﻿using Fams3Adapter.Dynamics.Address;
+using Fams3Adapter.Dynamics.Person;
+using Fams3Adapter.Dynamics.PhoneNumber;
+using Fams3Adapter.Dynamics.SearchRequest;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Fams3Adapter.Dynamics.Employment
+{
+    public class SSG_Employment :  DynamicsEntity
+    {
+
```

Comment:
no phone yet?

Reply:
We can't add phone entity in this. Phone is a different entity in dynamics. We will handle the mapping with the existing phone entity in dynamics not within the employment entity